build.sh

```diff
@@ -139,6 +139,7 @@ function build_desktop_target {
     if [[ ! -d "CMakeFiles" ]] || [[ "$ISSUE_CMAKE_ALWAYS" == "true" ]]; then
         cmake \
             -G "$BUILD_GENERATOR" \
+            -DIMPORT_EXECUTABLES_DIR=out \
```

Comment:
Are you sure it's `out` and not `.` or `..`? We're already in the `out/cmake-xxxx` directory here.